packages/lodestar/src/api/impl/validator/validator.ts

```diff
@@ -235,11 +235,14 @@ export class ValidatorApi implements IValidatorApi {
       return current;
     });
 
-    return {
+    const aggregateAndProof = {
       aggregate,
       aggregatorIndex,
       selectionProof: EMPTY_SIGNATURE,
     };
+
+    await this.db.seenAttestationCache.addAggregateAndProof(aggregateAndProof);
```

Comment:
I think this should go in publishAggregateAndProof

Reply:
done